lib/pre-commit/template.rb

```diff
@@ -0,0 +1,83 @@
+require "date"
+require "erb"
+require "fileutils"
+
+module PreCommit
+  class Template
+    TEMPLATE_DIR = File.expand_path("../../../templates/gem", __FILE__)
+
+    attr_reader :name, :author, :email, :description, :gem_name, :copyright
+
+    def initialize(*args)
+      @name, @author, @email, @description = args
```

Comment:
I'm really unhappy with this, it's good for start, but later on it would make sense to allow saving `@author, @email` in configuration and only if missing require them maybe as last param in form `'Name <email@...>'`